### ISSUE TITLE
the synset.txt in  pretrained resnext-101 model is wrong

### ISSUE THREAD

**Issue by li-haoran** [apache/incubator-mxnet#5557]
the synset.txt in [resnext](http://data.mxnet.io/models/imagenet/resnext/) is wrong order. i generate a new [synset.txt](https://github.com/li-haoran/imagenet-prediction) by using the train.lst

**Comment by piiswrong** [comment]
@mli @tornadomeet

**Comment by tornadomeet** [comment]
@terrychenism could your check it, which you trained the  model:`resnext-101`?

**Comment by terrychenism** [comment]
Thanks @li-haoran  I use imagenet'15 label to train resnext-101, different from resnet-101 which use imagenet'12 label.  I will update the synset.txt quickly. 
@piiswrong @tornadomeet @mli Do we need to retrain the resnext model, 1. match the torch result and use image io api, 2. match resnet label

**Comment by tornadomeet** [comment]
@terrychenism  i suggest update the synset.txt in http://data.mxnet.io/models/imagenet/resnext first, because the current resnext-101 model is ok. 
but a model with more accurate result and  the same synset.txt  is welcome if have time and resource。

**Comment by terrychenism** [comment]
Yes, the synset.txt has been updated.

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
